Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -140,8 +140,26 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
 
     /// Attempts to sends the current forms information to GitHub, on success will redirect the user to the new issue
     @objc func onSend() {
+        if let bodyText = bodyText {
+            self.finishSend()
+        } else {
+        let submitAlert = UIAlertController(title: "Please Provide Description", message: "Are you certain you want to submit this issue without a description?", preferredStyle: UIAlertControllerStyle.alert)
+
+        submitAlert.addAction(UIAlertAction(title: "Submit", style: .default, handler: { (action: UIAlertAction!) in
```

Comment:
I think you can just do `handler: { _ in ...
}`

Reply:
Instead of all the crazy ellipses? That would be cool.

Reply:
Nope. I still count all those crazy ellipses. But I was able to clean everything up overall.

Reply:
Can you check again? You should be able to change this:

```diff
- { (action: UIAlertAction!) in
+ { _ in
```